test/gtest/ucm/malloc_hook.cc

```diff
@@ -829,12 +829,13 @@ UCS_TEST_F(malloc_hook, dlopen) {
                                    reinterpret_cast<void*>(this));
     ASSERT_UCS_OK(status);
 
-    lib_name  = (char*)malloc(strlen(GTEST_UCM_HOOK_LIB_DIR) + strlen(libdlopen_test) + 1);
-    lib2_name = (char*)malloc(strlen(GTEST_UCM_HOOK_LIB_DIR) + strlen(libdlopen_test2) + 1);
-    strcat(strcpy(lib_name, GTEST_UCM_HOOK_LIB_DIR), libdlopen_test);
-    strcat(strcpy(lib2_name, GTEST_UCM_HOOK_LIB_DIR), libdlopen_test2);
+    lib_load = std::string(GTEST_UCM_HOOK_LIB_DIR) + libdlopen_load;
+    lib_mmap = std::string(GTEST_UCM_HOOK_LIB_DIR) + libdlopen_mmap;
 
-    lib = dlopen(lib_name, RTLD_NOW);
+    UCS_TEST_MESSAGE << lib_load;
```

Comment:
need to print also what it is (e.g "Loading " << lib_load), or remove the message

Reply:
added "Loading"